### PR TITLE
Update ragone plot measurement

### PR DESCRIPTION
**Pull request by dalg24**
moving to `Experiment` interface

### CODE REVIEW

File: python/source/ragone_plot.py [hunk 1]
```diff
@@ -151,3 +154,93 @@ def retrieve_performance_data(fin):
...
+                        break
+                    else:
+                        time_step = data['time'][-1]
+                        time_step /= self._max_steps_per_discharge
+                        self._ptree.put_double('time_step', time_step)
+                                               
+            except RuntimeError:
+                print('Failed to discharge at {0} watt'.format(discharge_power))
+                break
+            # TODO:
```
Comment (Rombur) [inline python/source/ragone_plot.py]: What's left to do here?

**Comment by Rombur** [comment]
Fix the comment and merge yourself

**Comment by dalg24** [comment]
This is a WIP ;)

**Comment by dalg24** [comment]
still need to remove `measure_performance`.
can't use lambda function in the verification like I did for EIS because I defined intermediary variable in the exact solution :(

**Comment by dalg24** [comment]
should we reset `time_step` in `RagoneAnalysis`? [more a memo for myself than a question]

**Pull request merged**